programmers/이상한-문자-만들기.js

```diff
@@ -0,0 +1,15 @@
+function solution(s) {
+  const answer = [];
+  let count = 0;
+  
+  for (let i = 0; i < s.length; i++) {
+    if (s[i] === " ") {
+        count = 0;
+        answer.push(' ');
+        continue;
+    }
+    if (count % 2 == 0 ) answer.push(s[i].toUpperCase());
+    else  answer.push(s[i].toLowerCase());
+   }
```

Comment:
현재  count를 증감시키는 코드가 없어서 전부 대문자로 출력되고있음. 다시 한번 확인했으면 좋겠고continue도 제거했음. 

```suggestion
  for (let i = 0; i < s.length; i++) {
    if (s[i] === " ") {
        count = 0;
        answer.push(' ');
    } else {
        if (count % 2 === 0) answer.push(s[i].toUpperCase());
        else answer.push(s[i].toLowerCase());
        
        count++;
    }
  }
```